lib/rust/parser/src/macros/resolver.rs

```diff
@@ -438,7 +438,6 @@ impl<'s> Resolver<'s> {
             self.replace_current_with_parent_macro(parent_macro);
             Step::MacroStackPop(token.into())
         } else if let Some(segments) = root_macro_map.get(repr) {
-//        } else if let Some(segments) = root_macro_map.get(repr) && token.variant.can_start_macro() {
```

Comment:
I wasn't sure which line to keep...

Reply:
Your choice was correct, thanks for handling that merge. And thanks for fixing my too-fancy-for-the-compiler attempt at downcasting, I think `instanceof` is probably the more idiomatic implementation there anyway.